integration-tests/bats/remotes.bats

```diff
@@ -2104,6 +2104,77 @@ SQL
     [[ "$output" =~ "branch 'feature3' set up to track 'origin/other'" ]] || false
 }
 
+@test "remotes: call dolt_branch track flag sets upstream" {
+    mkdir remote
+    mkdir repo1
+
+    cd repo1
+    dolt init
+    dolt remote add origin file://../remote
+    dolt sql -q "CREATE TABLE a (pk int)"
+    dolt add .
+    dolt commit -am "add table a"
```

Comment:
(super minor) if you capitalize the `a` you could delete line 2115. always nice to save a line of code! 😉 

```suggestion
    dolt commit -Am "add table a"
```